src/Engines/webgpuEngine.ts

```diff
@@ -2030,7 +2031,8 @@ export class WebGPUEngine extends Engine {
             gpuTextureWrapper = this._textureHelper.createGPUTextureForInternalTexture(texture, width, height);
         }
 
-        createImageBitmap(canvas).then((bitmap) => {
+        const engine = EngineStore.LastCreatedEngine;
+        engine && engine.createImageBitmap(canvas).then((bitmap) => {

```

Comment:
ditto

Reply:
You should be able to simply do `this.createImageBitmap(...)`?